workers/go/throughputstress/workflow.go

```diff
@@ -248,7 +249,7 @@ func nexusClient(ctx workflow.Context, params *throughputstress.WorkflowParams)
 		workflow.GetLogger(ctx).Debug("not running nexus tests, set nexusEndpoint in options to enable these tests")
 		return nil
 	}
-	return workflow.NewNexusClient(params.NexusEndpoint, ThroughputStressServiceName)
+	return workflow.NewNexusClient(params.NexusEndpoint, kitchensink.KitchenSinkServiceName)
```

Comment:
This whole workflow will disappear shortly; no need to maintain two Nexus services/names.

Reply:
🥳